docs/content/why-temporal.md

```diff
@@ -0,0 +1,167 @@
+---
+id: why-temporal
+title: What are the differences between Temporal and a traditional system?
+description: This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import CenteredImage from "../components/CenteredImage.js"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../content/what-is-a-workflow-execution.md'
+import * as WhatIsASignal from '../content/what-is-a-signal.md'
+import * as WhatIsAQuery from '../content/what-is-a-query.md'
+
+One of the aspects of the Temporal System is that it abstracts the complexity of a distributed system.
+Distributed systems exist to scale computation across multiple machines as the potential load of a system changes.
+In theory a distributed system facilitates a reliable and highly performant application.
+
+However there are a set of failures that can occur in a distributed application that can make things very complicated, especially at a large scale.
```

Comment:
```suggestion
However, a set of failures can occur in a distributed application that can make things very complicated, especially at a large scale.
```

Reply:
```suggestion
However, a set of failures can occur in a distributed application that can make things very complicated, especially at a large scale.
```
just seeing if it will let me commit my suggestion - as it wont let me commit yours.